data/seed/Seeding.md

```diff
@@ -42,8 +42,8 @@ $ python manage.py loaddata data/seed/2024-03-15-foilaw.json
 ```
 
 > NOTE: The Classifications data has been extracted from the Classifications field in
-> the extract of public bodies from the production UIPA.org website on
-> 03/15/2024. See 2024-03-15-Hawaii_UIPA_Public_Bodies_All.csv.
+> the extract of public bodies from the production UIPA.org website.
+> You will need to update the json file date to the current set in data/seed.
```

Comment:
I think these lines shouldn't be changed - I think lines 39-41 should be changed instead.